TCAT/AppDelegate.swift

```diff
@@ -115,6 +115,36 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
     }
     
+    @objc func storeServicedRoutes() {
+        Network.getAlerts().perform(withSuccess: { (alertRequest) in
+            if alertRequest.success {
+                if let currentRoutes = self.userDefaults.array(forKey: Constants.UserDefaults.servicedRoutes) as? [Int] {
+                    let alertRoutes = self.allRoutesInAlerts(alertRequest.data)
+                    if !alertRoutes.containsSameElements(as: currentRoutes) {
+                        self.userDefaults.set(alertRoutes, forKey: Constants.UserDefaults.servicedRoutes)
+                    }
+                } else {
+                    let alertRoutes = self.allRoutesInAlerts(alertRequest.data)
+                    self.userDefaults.set(alertRoutes, forKey: Constants.UserDefaults.servicedRoutes)
+                }
+            } else { print("storeServicedRoutesError: Network request failed" ) }
+        }) { (error) in
+            print("storeServicedRoutesError: \(error.localizedDescription)")
```

Comment:
Should we fail with a UIAlert, similar to the Couldn't Get Stops alert message? Maybe check with design